liblocation/src/main/java/com/mapbox/android/core/location/LocationEngineResult.java

```diff
@@ -55,7 +55,7 @@ public static LocationEngineResult create(List<Location> locations) {
    * @since 3.0.0
    */
   public Location getLastLocation() {
```

Comment:
nit: it would be great to add `@Nullable` annotation here.

Reply:
@yunikkk good catch! please continue keeping an eye on our PRs ;) I'll raise PR with the fix shortly.